pkg/api/controllers/swagger.yaml

```diff
@@ -1016,7 +1016,12 @@ components:
       type: object
       nullable: true
       additionalProperties: {}
-      example: { admin: true, a: { nested: { key: value}} }
+      example: { admin: true, a: { nested: { key: value } } }
+    AdditionalOperations:
+      type: object
+      nullable: true
+      additionalProperties: {}
+      example: { set_account_meta: { users:001: { a: { nested: { key: value } } } } }
```

Comment:
do you think we could maybe use "set_account_metadata" instead ?

Reply:
In the future Numscript feature, we want to have the new instruction as `set_account_meta`, because we already have the `set_tx_meta` instruction. I wanted to have the same naming for both :)